### ISSUE TITLE
Insight API seems to be down

### ISSUE THREAD

**Issue by albmunmu** [guerrerocarlos/bitcoin-receive-payments#16]
Hello

I am getting the following error when using the gateway:

`UnhandledPromiseRejectionWarning: Cannot reach any bitcoin insight server... no bitcoin transactions are being received.`

Checking the code I can see the insight servers are these ones: ["https://insight.bitpay.com/api/", "https://www.localbitcoinschain.com/api/", "https://search.bitaccess.co/insight-api/"]

They all seem to be down, are there any alternatives for these APIs ?

Thank you for your great work!

**Comment by ThomsBaum** [comment]
Hey sir, have you find a solution?

**Comment by zkinion** [comment]
I think this project is dead and the chances of the owner coming in here and solving anything are slim to none.

**Comment by guerrerocarlos** [comment]
You are right, existing APIs used by the module are down, looking for alternatives.
